src/components/Book.css

```diff
@@ -0,0 +1,184 @@
+.Lesson-Panel {
+  width: 60rem;
+  height: 10.625rem;
+  margin: 0.938rem 0.063rem 2.5rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  display: flex;
+}
```

Comment:
- [ ] Please fix the left margins and make sure that your design matches the Zeplin design.

### Your design
![image](https://user-images.githubusercontent.com/83803414/192111237-195c2111-7a70-46fd-9275-fe529f4d2cb1.png)
![image](https://user-images.githubusercontent.com/83803414/192111233-5161f0b3-77f5-4999-a7c7-5364b980ca75.png)

### Zeplin design

![image](https://user-images.githubusercontent.com/83803414/192111266-7bbcc1c1-4d08-4c96-a3c0-92b6eaaa67d0.png)
![image](https://user-images.githubusercontent.com/83803414/192111277-0b199ca2-5e41-4fe9-bfef-e5243fc96a8a.png)